getting-started/config/hyper_parameter_tuning/toy_example_v2/mappo_rnn_road_env.yaml

```diff
@@ -52,7 +52,8 @@ REWARD_STANDARDIZATION: True
 "WANDB_LOG_ALL_SEEDS": False # will log vmapped seeds separately
 
 # where to save the params (if None, will not save)
-"SAVE_PATH": null
+IF_SAVE: False
```

Comment:
Why do we include two variables (instead of the default option `if config.get("SAVE_PATH", None) is not None:` )? Now we have to modify two inputs in the config.